modules/samtools_sort_index.cfmod.pl

```diff
@@ -114,8 +114,8 @@
 		warn "Parameter 'forcesort' set, so not checking BAM to see if it's sorted..\n";
 	}
 
-	$cfcores = $cf{'cores'};
-	$command = "samtools sort -m $mem_per_thread -@ $cfcores $namesort $sortfile -o $output_fn";
+	my $cfcores = $cf{'cores'};
+	my $command = "samtools sort -m $mem_per_thread -@ $cfcores $namesort -o $output_fn";
 	warn "\n###CFCMD $command\n\n";
```

Comment:
why did you take out the $sortfile? isn't this required?

Reply:
Oops, yes. Should just be `$file` though I guess, as the `samtools view` part isn't needed any more for BAM compatibility.